src/main/java/co/rsk/federate/btcreleaseclient/BtcReleaseClient.java

```diff
@@ -160,10 +161,21 @@ public void setup(
     }
 
     public void start(Federation federation) {
+       FederationMember federationMember = federatorSupport.getFederationMember();
+       if (federation.isMember(federationMember)) {
```

Comment:
Should this condition be negative? `if (!isMember)` 😳

Isn't this covered by a test?